pegasus/helpers.rb

```diff
@@ -14,11 +14,6 @@ def authentication_required!(url=request.url)
   redirect((request.scheme || 'http') + ':' + CDO.studio_url("/users/sign_in?return_to=#{url}"), 302)
 end
 
-def owns_channel?(encrypted_channel_id)
```

Comment:
Great idea to extract `owns_channel?` here. I added this implementation to pegasus in a different PR in response to your earlier feedback on this PR, and now I'm moving it to where I can access it from `shared/`.